### DISCUSSION TITLE
Taskbar where window is open

### DISCUSSION THREAD

**Discussion by HigeIris** [valinet/ExplorerPatcher#761] [Q&A]
Hello,

Thanks a lot for your great tool, it's perfect for me to work properly on win11..

I just add this thread for people like me who are looking for this option. It takes me almost an hour to find it, because I'm not searching it in the right settings...

As you can see on the picture bellow, the settings window application appears on the taskbar of the screen 1 and 2, but the application is only open on the screen 2
![image](https://user-images.githubusercontent.com/56878082/151799020-2511251a-8534-4c40-9c79-9708a61bfcea.png)

To configure it, there is not direct option in ExplorerPatcher. You can found it in Windows settings→Personalization→Taskbar→Taskbar behaviors (could be open in one click, using the option of ExplorerPatcher→Taskbar→More taskbar option in the Settings app or right click on the taskbar→Taskbar settings).

Set the combobox "When using multiple displays, show my taskbar to show the desktop" to "Taskbar where window is open" (see picture below)
![image](https://user-images.githubusercontent.com/56878082/151800445-b3287be5-2a39-4571-8184-cf33e182ec7e.png)

That's it :)

**Comment by valinet** [comment]
Yeah, initially the idea was not to add options that duplicate the "Taskbar settings" section in the Settings app. Nowadays, for the sake of completeness and having everything in one place, I may actually considering adding an option for that in the future.
